_posts/2020-03-15-learning-to-share-accomplishments-1.md

```diff
@@ -0,0 +1,96 @@
+downright terrible. It just felt wrong almost to try to write about all these things. It felt like bragging.
+
+The easiest way I found to toot my own horn? I found one rule makes it easy: **Stick to the facts.**
+
+The basics are to look at things I actually accomplished factually. These things definitively are provable. They have 
+a place, a time, a number, or just anything that is solid and concrete about them. These aren't disputable because 
+they're not my opinions. I _did_ work that job. I _was_ on that team. I _wrote_ that code for _that_ project. Because 
+I can't dispute these, it makes it easier to say about myself.
+
+But the basics of a bio are easy. Actually taking a team accomplishment and diving into my specific thing is a bit 
+harder (at least mentally).
+
+For each project/team/accomplishment, what was my one specific contribution? In some cases this is easy, like when I 
+implemented a feature for a project that made it into production. Other times less so, like when I built a thing and 
+learned it later got scrapped. The scrapping wasn't my fault, and it feels weird to talk about it because it isn't 
+visible anymore. It's still an accomplishment of mine though, and it's one of the things I'm trying to keep an open 
+mind about when writing this post.
+
+Finally, what specifics can I back up things with? Did something I build save the team time? Did it save the company 
+money? Did it solve a major crisis? Did it fix a security flaw? These go back to the thing about adding more facts. If 
+the company said "We think we can save this much money by running this in production", that's a thing I can definitely 
+throw in the mix.
+
+"But Sarah," you may say, "the impostor syndrome! I don't really deserve that credit! I wasn't the only one who worked 
+on that thing!" (Or insert your favorite excuse here.) Tough cookies. Write it down anyway. Just because you weren't 
+the only one that worked on it didn't mean _you_ didn't work on it. Keep that in mind! (And this is also where I need 
+to take my own advice a bit more.)
+
+Over time I think adding all these little things up helps make me sound awesome, even though individually maybe they 
+don't. I don't think I'm born a level of more awesome than anyone else, I just happened to do a few cool things that, 
+when bundled together in one narrative, have made some people say that I'm awesome. I really think anyone can probably 
+do the same for their own interviews or bios.
+
+## What I Personally Have Worked on in My Past Jobs
+
+I wanted to spend some time looking back at my different jobs and the work I have done. I decided to put that into a 
+separate post. Feel free to check that out [Part 2](../learning-to-share-accomplishments-2/).
+ 
+## Conclusion
+
+As I continue to interview at places, a cloud of insecurity has been starting to come over me. I've felt like because 
```

Comment:
I think a lot of people are going to appreciate your venerability here because damn we are all thinking it. It can be so hard <3